pkgs/servers/rpiplay/default.nix

```
@@ -0,0 +1,49 @@
+{ lib, stdenv, pkg-config, fetchFromGitHub, fetchpatch, cmake, wrapGAppsHook, avahi, avahi-compat, openssl, gst_all_1, libplist }:
+
+stdenv.mkDerivation rec {
+  pname = "rpiplay";
+  version = "unstable-2021-04-29";
```

```suggestion
  version = "unstable-2021-02-27";
```

This is supposed to be the commit date, not the date of initially fetching.